<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileView</class>
 <widget class="QMainWindow" name="ProfileView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="rawLab">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="loadRawLab">
        <property name="text">
         <string>Labor Laden</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="copyAllLabs">
        <property name="text">
         <string>Alle Labore kopieren</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearAll">
        <property name="text">
         <string>neuer Patient</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="delimiterCombo">
        <item>
         <property name="text">
          <string>Komma</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Punkt</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="labTable">
      <property name="contextMenuPolicy">
       <enum>Qt::CustomContextMenu</enum>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::EditKeyPressed</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
